content/_index.md

```diff
@@ -4,13 +4,14 @@ hide_title: true
 images:
 - https://odin-lang.org/images/logo-slim.png
 ---
-<center id="home-info">
-<a href="//odin-lang.org">
-	<div style="max-width: 20em"><img alt="Odin Programming Language" src="/images/logo-slim.png"></div>
-</a>
+<center id="hero">
```

Comment:
I try to make all my sites usable without CSS so I try never to use `<center>` if I can. Try to make this centred in CSS instead.

Reply:
It was already center before I even touched it. I just added the hero id.

Reply:
I know. That's a mistake on my end too, which I noted in a later comment.